src/nvim/api/vim.c

```diff
@@ -135,7 +137,6 @@ Dictionary nvim__get_hl_defs(Integer ns_id, Error *err)
 ///                               same as attributes of gui color
 /// @param[out] err Error details, if any
 ///
-/// TODO: ns_id = 0, should modify :highlight namespace
 /// TODO val should take update vs reset flag
```

Comment:
While you're at it, you could move this out of the docstring to avoid putting the TODO in the documentation.

Reply:
I did find the first (now removed) TODO pretty valuable in figuring out why `nvim_set_hl` wasn't visible in the output of `:highlight<cr>` which is the main tool I use for debugging highlights. Though I agree, the current format isn't ideal. However, the second TODO you're referencing isn't super useful here anyway.

Reply:
Ideally, that should have been a part of the actual documentation ("Note: currently can't set global namespace" or somesuch) instead of just a raw `TODO` comment... (I don't believe it was meant to actually appear in the documentation.)